include/ddc/kernels/splines/spline_builder.hpp

```diff
@@ -890,4 +929,125 @@ operator()(
                 });
     }
 }
+
+template <class OutMemorySpace>
+template <
+        class ExecSpace,
+        class MemorySpace,
+        class BSplines,
+        class InterpolationDDim,
+        ddc::BoundCond BcLower,
+        ddc::BoundCond BcUpper,
+        SplineSolver Solver,
+        class... IDimX>
```

Comment:
```suggestion
template <
        class ExecSpace,
        class MemorySpace,
        class BSplines,
        class InterpolationDDim,
        ddc::BoundCond BcLower,
        ddc::BoundCond BcUpper,
        SplineSolver Solver,
        class... IDimX>
template <class OutMemorySpace>
```

Reply:
Any idea how to get doxygen to recognise it when it's this way round?

Reply:
Could it be related to the default value of the template parameter ?

Reply:
It doesn't seem to be. Adding the default value doesn't make a difference. Maybe the definitions below the class should be in a `/// \cond` region?

Reply:
There was a mismatch of templates between declaration and definition.